.github/workflows/very_good_cli.yaml

```diff
@@ -50,9 +50,15 @@ jobs:
 
     strategy:
       matrix:
+        flutter-version:
+          # The version of Flutter to use should use the minimum Dart SDK version supported by the package,
+          # refer to https://docs.flutter.dev/development/tools/sdk/releases.
+          - "3.3.0"
+          - ""
```

Comment:
We should test with only one flutter version. Things like `dart format` can break in between flutter versions and there is no way to support both.